mevolib/inference/__init__.py

```diff
@@ -169,15 +186,15 @@ def get_phylogeny ( binary, infile, infile_format, args = 'default',
     command = [binary] + gen_args(args, infile_path, bootstraps)
     # Run the phylogenetic inference process handling any Runtime exception
     try :
-        output = subprocess.check_output(command, stderr=DEVNULL,
+        output = subprocess.check_output(command, stderr=subprocess.DEVNULL,
```

Comment:
`subprocess.check_output()` is no longer the recommended approach, but `subprocess.run()` (with the corresponding arguments).